clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1047,7 +1096,7 @@ class SyclKernelFieldHandler {
 class SyclKernelFieldChecker : public SyclKernelFieldHandler {
   bool IsInvalid = false;
   DiagnosticsEngine &Diag;
-
+  static constexpr const bool VisitUnionBody = false;
```

Comment:
This line should be in SyclKernelFieldHandler, not here.

Reply:
Done